extmod/uasyncio/event.py

```diff
@@ -23,7 +23,7 @@ def set(self):
     def clear(self):
         self.state = False
 
-    async def wait(self):
+    def wait(self):
```

Comment:
I think we should add a single-line comment to let users know this is actually an async function (a leaf async function) that needs to be await'd.  Eg:
```py
# async
def wait(self):
    ...
```

Reply:
(and please add the same comment to all functions in this PR that had the `async` removed)

Reply:
Sounds reasonable. Done.